Tests/CommandsTests/BuildCommandTests.swift

```diff
@@ -190,48 +202,6 @@ class BuildCommandTestCases: CommandsBuildProviderTestCase {
         }
     }
 
-    func testBinSymlink() async throws {
```

Comment:
suggestion: In order to ensure the "concrete" class  implement this this, can we keep an implementation that will fail, forcing the subclasses to implement the test?

This way, we are guaranteed that each sub-class will implement this test.